app/controllers/warehouse_reports/hud/not_one_hohs_controller.rb

```diff
@@ -35,7 +35,8 @@ def index
 
       respond_to do |format|
         format.html do
-          @enrollments = @enrollments.page(params[:page]).per(50)
+          @enrollments = @enrollments.page(params[:page]).per(25)
+          @enrollmnts_array = @enrollments.to_a
```

Comment:
Typo: `@enrollmnts_array`

Reply:
Also, a comment explaining this seems worthwhile, because it *looks* unnecessary.